commands/password-managers/bitwarden/copy-first-matching-password.sh

```diff
@@ -49,12 +49,12 @@ item=$(bw list items --search $1 $session 2> /dev/null | jq ".[0] | { name: .nam
 name=$(echo $item | jq ".name")
 password=$(echo $item | jq --raw-output ".password")
 
-if [[ -z $name || -z $password ]]; then
+if [[ -z $name || -z $password || ( $name == "null" && $password == "null") ]]; then
   echo "The query '$1' did not return a password."
   exit 1
 fi
 
-echo $password | pbcopy
+echo $password | tr -d " \t\n\r" | pbcopy
```

Comment:
Per the [`jq` documentation](https://stedolan.github.io/jq/manual/#Invokingjq), we should probably be using `--join-output` instead of `--raw-output` on L50, instead of trying to script the `\n` away like this. I didn't realize `--join-output` was an option when I originally wrote this script.

Reply:
I did investigate a bit, and I finally found that the added newline isn't caused by the `jq` command but by the `echo` one.
I think the following code would be the cleanest way to resolve this to avoid the added newline at the end from the `echo` command as documented [here](https://linuxcommand.org/lc3_man_pages/echoh.html).
```suggestion
echo -n $password | pbcopy
```

Reply:
Good looks! Yes, let's go with `echo -n`.